libs/order/src/guards/placed-order/placed-order-guard-redirect.token.ts

```diff
@@ -0,0 +1,3 @@
+import { InjectionToken } from '@angular/core';
+
+export const DaffOrderPlacedOrderGuardRedirectUrl = new InjectionToken<string>('DaffOrderPlacedOrderGuardRedirectUrl');
```

Comment:
I'd think this would be called "DaffPlacedOrderGuardRedirectUrl"

Reply:
changed